application/blocks/list-block.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace CNET\Bridge\Blocks;
+
+class List_Block extends Block_Parser implements Block_Interface {
+
+	const BLOCK_TYPE = 'core/list';
```

Comment:
should we have a constant class that contains all of them? Or do you think this is good enough?